extensions/amp-story/1.0/amp-story-share-menu.js

```diff
@@ -156,6 +156,7 @@ export class ShareMenu {
    */
   buildForFallbackSharing_() {
     const root = this.win_.document.createElement('div');
+    root.className = 'i-amphtml-story-share-menu-host';
```

Comment:
Please use the classList API otherwise you're removing all the other classes already set

Reply:
These shadow roots didn't have any classes, but will add them to avoid running into issues like the one you mentioned in the future.